### ISSUE TITLE
Stimulus 2.0.0 compatibility

### ISSUE THREAD

**Issue by miketheman** [afcapel/stimulus-autocomplete#27]
Hello!

Stimulus 2.0.0 changed the default behavior of the `data-target` attribute, and now trigger warnings in the JS Console to surface the deprecation.

However, after updating to the 2.0.0 version, pages that use this package throw this warning:

```
Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at ValueObserver.get (value_observer.js:73)
    at ValueObserver.invokeChangedCallbacksForDefaultValues (value_observer.js:48)
    at new ValueObserver (value_observer.js:11)
    at new Context (context.js:12)
    at Module.fetchContextForScope (module.js:55)
    at Module.connectContextForScope (module.js:37)
    at Router.scopeConnected (router.js:114)
    at ScopeObserver.elementMatchedValue (scope_observer.js:54)
    at ValueListObserver.tokenMatched (value_list_observer.js:54)
```

I'm not sure how to resolve this yet, but wanted to log the finding in case someone else has. I've confirmed reverting to Stimulus 1.1.1 returns the app to previous behavior.

**Comment by Intrepidd** [comment]
See https://github.com/afcapel/stimulus-autocomplete/pull/26

**Comment by drnic** [comment]
Now merged to master via #30 (and #26)
